File: JavaScript/bubbleSort.js
class bubbleSort {
    sort(intArray) {
        if (typeof (intArray) === 'Array') {
            const size = intArray.size;
            let unSorted = size;
            for (let i = 0; i < size; i++) {
                for(let j=1;j<unSorted;j++){
                    if(intArray[j-1]>intArray[j]){
                        this.swap(intArray,j-1,j);
                    }
                }
                unSorted--;
            }

        }
        return intArray;
    }
    swap(intArray,i,j){
        const temp =intArray[i];
        intArray[i]=intArray[j];
        intArray[j]=temp;
    }
}

const bubble = new bubbleSort();
const result= bubble.sort([-22, -15, 1, 7, 20, 35, 55]);
console.log(result);

File: C#/Algorithms.Implementation/BubbleSort/BubbleSort.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Algorithms.Implementation.BubbleSort
{
    public class BubbleSort
    {
        public int[] Sort(int[] intArray)
        {
            if (intArray == null) throw new ArgumentNullException("Int Array", "don't allow to you pass null value for this Method");
            int size = intArray.Length;
            int unSortedLength = size;
            for (int i = 0; i < size; i++)
            {
                for (int j = 1; j < unSortedLength; j++)
                {
                    if (intArray[j - 1] > intArray[j])
                    {
                        Swap(intArray, j - 1, j);
                    }
                }
                unSortedLength--;
            }
            return intArray;
        }

        private void Swap(int[] intArray, int v, int j)
        {
            int temp = intArray[v];
            intArray[v] = intArray[j];
            intArray[j] = temp;
        }
    }
}
